From patchwork Thu Apr 22 04:30:56 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 12217625
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-10.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_PATCH,
	MAILING_LIST_MULTI,SPF_HELO_NONE,SPF_PASS autolearn=unavailable
	autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 9EDDEC433ED
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 04:31:05 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 6EEA661222
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 04:31:05 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S230119AbhDVEbi (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 00:31:38 -0400
Received: from ozlabs.org ([203.11.71.1]:55037 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S229561AbhDVEbh (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 00:31:37 -0400
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 4FQkxs3vqCz9sWK;
        Thu, 22 Apr 2021 14:30:57 +1000 (AEST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1619065861;
        bh=nrrjueQbJLgdff/uUfD6SBTPY8DEJYTf7uaFtxCIUK8=;
        h=Date:From:To:Cc:Subject:From;
        b=b8b+zTluZd0z2JjIQqelSzr8ILEsdpgJjpF09qTJnnzZhZNez7gYhy59bNjcc2mR2
         7tjj/amfOiKxjHvYYGfTD5uCBXYONp6ex1SzINLXcf0Hp2aLdHX/04o63MG+sDW5dk
         RTtiHfAxaaI3cXXKnQb6voyUtvpR9J6gNYplD/S0DpRsXKACHBylKH8+3lqTzgZ0O5
         EnYt0Vkd7Ffvyv3OcH6fJgmYDycMvR4WeB7u8QpYqtrneUrWYR73PsRsFQyO3TUQZT
         WWihwCjRWX+rJfjtn0oU0H5hGbqgxlgrbIfq+kHhLspz0tBodKSR7H6tYwlCFUlFN7
         ZKSGDxSPkzJpg==
Date: Thu, 22 Apr 2021 14:30:56 +1000
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paolo Bonzini <pbonzini@redhat.com>, KVM <kvm@vger.kernel.org>,
        Thomas Gleixner <tglx@linutronix.de>,
        Ingo Molnar <mingo@elte.hu>, "H. Peter Anvin" <hpa@zytor.com>,
        Peter Zijlstra <peterz@infradead.org>
Cc: Ingo Molnar <mingo@kernel.org>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        Linux Next Mailing List <linux-next@vger.kernel.org>,
        Nadav Amit <namit@vmware.com>,
        Wanpeng Li <wanpengli@tencent.com>
Subject: linux-next: manual merge of the kvm tree with the tip tree
Message-ID: <20210422143056.62a3fee4@canb.auug.org.au>
MIME-Version: 1.0
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm tree got a conflict in:

  arch/x86/kernel/kvm.c

between commit:

  4ce94eabac16 ("x86/mm/tlb: Flush remote and local TLBs concurrently")

from the tip tree and commit:

  2b519b5797d4 ("x86/kvm: Don't bother __pv_cpu_mask when !CONFIG_SMP")

from the kvm tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc arch/x86/kernel/kvm.c
index 5d32fa477a62,224a7a1ed6c3..000000000000
--- a/arch/x86/kernel/kvm.c
